dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/AuthorizationRestRepository.java

```diff
@@ -168,11 +193,69 @@ public AuthorizationRest findOne(Context context, String id) {
             context.switchContextUser(user);
         }
 
+        List<Authorization> authorizations =
+                findAuthorizationsByUUIDList(context, type, uuidList, user, featureNames);
+
+        if (currUser != user) {
```

Comment:
Again, seems like this should use the same equality check as above... `ObjectUtils.notEqual()`, unless I'm misunderstanding something